### PR TITLE
Add acceptance test for creating new files

### PR DESCRIPTION
**Pull request by jasson99**
## Description

Added acceptance tests for creating new files

## How Has This Been Tested?

:robot: 
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [x] Tests

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [x] Acceptance tests added
- [ ] Documentation ticket raised: <link> 

## Open tasks:

- [ ] ...

### CODE REVIEW

**Comment by jasson99** [comment]
cc please review @individual-it , @phil-davis

File: tests/acceptance/features/webUIFiles/createFile.feature [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    Given user "user1" has been created with default attributes
+    And user "user1" has logged in using the webUI
+    And the user has browsed to the files page
+
+  Scenario: create a new file
+    When the user creates a new file "sample.txt" using the webUI
+    Then file "sample.txt" should be listed on the webUI
+
+  Scenario: try to create a file that already exists
+     When the user tries to create a file with already existing name "lorem.txt" using the webUI
```
Comment (phil-davis) [inline tests/acceptance/features/webUIFiles/createFile.feature]: This is indented 1 char too much.

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>Files</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/phoenix/5939/

<p>





</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>Files</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/phoenix/5947/

<p>





</p></details>

**Pull request merged**